spark/src/test/java/io/ddf/spark/etl/TransformationHandlerTest.java

```diff
@@ -132,7 +132,7 @@ public void testReservedFactor() throws DDFException {
     ddf.setAsFactor("month");
```

Comment:
Ah, users can do like this newddf.getSchemaHandler().setAsFactor(column).setLevels(levels);